common/testData/units/unitsTest.txt

```diff
@@ -1,7 +1,7 @@
 # Test data for unit conversions
```

Comment:
Add a comment, "DO NOT EDIT THIS FILE, it is autogenerated by GenerateUnitTestData.java"?